src/CSharp/EasyMicroservice.MemoryPack/EasyMicroservice.MemoryPack.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Rename the "EasyMicroservice" folder's name to "EasyMicroservices".
However please fix the namespaces.
Please move "EasyMicroservice.MemoryPack" to "EasyMicroservices.Serialization.MemoryPack"
Please create a Providers folder and add MemoryPack Provider to it.
you can check other projects like EasyMicroservices.Serialization.System.Text.Json and EasyMicroservices.Serialization.BinaryGo

Reply:
conversion solved